README.rst

```diff
@@ -26,9 +25,7 @@ learning projects:
 
 #. **Track** experiments in your local Git repo (no servers needed).
 
-#. **Compare** any data, code, parameters, metrics, or plots from the command
-   line or in `VS Code
-   <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_.
+#. **Compare** any data, code, parameters, modelhttps://www.youtube.com/watch?v=IYhVmD-_wRometrics, or performance plots
```

Comment:
What happened here 😄 ?

Reply:
Consistent formatting (in the places I edited). 1 sentence = 1 line

Reply:
Lol I just realized the hero rat video got in. Gone now